src/uct/tcp/tcp_sockcm.c

```diff
@@ -22,9 +23,18 @@ ucs_config_field_t uct_tcp_sockcm_config_table[] = {
    "TCP CM private data length",
    ucs_offsetof(uct_tcp_sockcm_config_t, priv_data_len), UCS_CONFIG_TYPE_MEMUNITS},
 
+   UCT_TCP_SEND_RECV_BUF_FIELDS(auto, ucs_offsetof(uct_tcp_sockcm_config_t, sockopt)),
+
   {NULL}
 };
 
+void uct_tcp_sockcm_close_fd(int fd)
+{
+    if (close(fd) < 0) {
```

Comment:
where is this function used?

Reply:
it's a leftover left by mistake, removing...